tests/export_test.py

```diff
@@ -311,6 +314,78 @@ def f(a_b_pair, a, b):
     self.assertAllClose(f((a, b), a=a, b=b),
                         exp_f.call((a, b), a=a, b=b))
 
+  def test_pytree_namedtuple(self):
+    T = collections.namedtuple("SomeType", ("a", "b", "c"))
+    export.register_namedtuple_serialization(
+        T,
+        serialized_name="test_pytree_namedtuple.SomeType",
```

Comment:
Could/Should this not be just the type __name__ and __module__? Otherwise you run the risk of multiple registrations with the same name.

Reply:
I was planning to leave up to the user how to best use that string identifier. E.g., it can be a module and name and version.

Reply:
I assume it's an error to register multiple handlers under the same name?

Reply:
Not at the moment, but I think this would be a good idea. I'll add it.